appveyor.yml

```diff
@@ -17,17 +17,15 @@ platform:
   - x64
 
 install:
-  - set "PATH=%PYTHON_DIR%;%PATH%"
-  - set "APPVEYOR_PYTHON=%PYTHON_DIR%/python.exe"
-  - python -m pip install -U pip tox wheel
```

Comment:
Thanks to @nicoddemus for finding out that we can't use this `python`, it's better to always call `python` with the full command-line